### ISSUE TITLE
Universal Programming Panel: Links do not have a focus indicator - 2.4.7: Focus Visible

### ISSUE THREAD

**Issue by syarwick23** [stephenravenscraft/loyola-template-4.0#104]
The link on the calendar date area does not have a visible focus indicator.

**Comment by syarwick23** [comment]
Issue fixed with new template
